cmd/peek-submission/main_test.go

```diff
@@ -60,6 +60,10 @@ func TestPeekBase(test *testing.T) {
 			continue
 		}
 
+		if testCase.expectedExitCode != 0 {
```

Comment:
You include test cases with non-zero exit codes that have locators (which makes sense). But here, we skip over the locator check for non-zero exit codes.

Since we want to include the check, we need to restructure these checks. Here is the API [version](https://github.com/edulinq/autograder-server/blob/e2448c4b3a9436ac441cf4f578a53de7b7054710/internal/api/courses/assignments/submissions/fetch_user_peek_test.go#L88).

Ignore this comment and remove the expected locator from the test struct IF a non-zero exit code prevents you from checking anything else.